test/tangents.jl

```diff
@@ -5,6 +5,12 @@
         (Cstring, NoTangent),
         (Cwstring, NoTangent),
         (Union{}, Union{}),
+        (Core.CodeInstance, NoTangent),
+        (Core.MethodInstance, NoTangent),
+        (Core.Binding, NoTangent),
+        (Core.Compiler.InferenceState, NoTangent),
+        (Core.Compiler.Timings.Timing, NoTangent),
+        (Core.Compiler.InferenceResult, NoTangent),
 
```

Comment:
missing `Core.Compiler.AnalysisResults`?

Reply:
Ah, see below :) It only exists on 1.11, so I've had to put it in a separate if statements.